doc/2/api/controllers/document/scroll/index.md

```diff
@@ -60,10 +60,12 @@ Returns a paginated search result set, with the following properties:
   - `_id`: document unique identifier
   - `_score`: [relevance score](https://www.elastic.co/guide/en/elasticsearch/guide/current/relevance-intro.html)
   - `_source`: new document content
-- `remaining`: remaining documents that can be fetched
 - `scrollId`: identifier to the next page of result. Can be different than the previous one(s)
 - `total`: total number of found documents. Usually greater than the number of documents in a result page
 
+<SinceBadge version="auto-version"/>
+- `remaining`: remaining documents that can be fetched
```

Comment:
It's better to put the badge on the same line
```suggestion
- `remaining`: remaining documents that can be fetched <SinceBadge version="auto-version"/>
```
![image](https://user-images.githubusercontent.com/4447392/89883073-f85c6100-dbc7-11ea-832f-52c093780fe7.png)

Reply:
Ah, it works if put at the end of the line, interesting... :+1: 

This looks a tad better than having a separate list, I agree.